src/LSST/M1M3/SS/Settings/HardpointActuatorSettings.cpp

```diff
@@ -46,17 +48,23 @@ void HardpointActuatorSettings::load(YAML::Node doc) {
         micrometersPerStep = doc["MicrometersPerStep"].as<double>();
         micrometersPerEncoder = doc["MicrometersPerEncoder"].as<double>();
 
-        auto _hpIntSettings = [doc](int32_t *data, const char *field) {
```

Comment:
This drops the functionality. The original load data from YAML Kafka version doesn't seems to do that.